### PR TITLE
TILA-2248 ext-link for recurring search reservation unit items

### PR DESCRIPTION
**Pull request by kieferbonk**

### CODE REVIEW

File: ui/components/search/ReservationUnitCard.tsx [hunk 1]
```diff
@@ -226,23 +220,38 @@ const ReservationUnitCard = ({
             ) : null}
           </Props>
           <Actions>
-            <div style={{ flexGrow: 1 }} />
             {containsReservationUnit(reservationUnit) ? (
               <MediumButton
-                iconLeft={<IconCheck />}
+                iconRight={<IconCheck />}
                 onClick={() => removeReservationUnit(reservationUnit)}
+                data-testid="reservation-unit-card__button--select"
```
Comment (joonatank) [inline ui/components/search/ReservationUnitCard.tsx:227]: Both buttons have same testid?

File: ui/components/search/ReservationUnitCard.tsx [hunk 2]
```diff
@@ -226,23 +220,38 @@ const ReservationUnitCard = ({
...
-            )}
+              </BlackButton>
+            )}{" "}
+            <Link
+              href={link}
+              target="_blank"
+              rel="noopener noreferrer"
+              style={{ display: "contents" }}
+            >
+              <BlackButton
```
Comment (joonatank) [inline ui/components/search/ReservationUnitCard.tsx:247]: Button inside a Link is suspicious.
Does <BlackButton as="span"... work for HDS components?

Comment (kieferbonk) [reply ui/components/search/ReservationUnitCard.tsx]: Handled link in button.

File: ui/components/search/ReservationUnitCard.tsx (hunk 1 above)
Comment (kieferbonk) [reply ui/components/search/ReservationUnitCard.tsx:226]: Well, they're semantically the same button.

**Review by joonatank — APPROVED** [review]

**Pull request merged**